osu.Game/GameModes/Menu/ButtonSystem.cs

```diff
@@ -50,42 +50,47 @@ public override void Load()
         {
             base.Load();
 
-            osuLogo = new OsuLogo(onOsuLogo)
+            Add(new Drawable[]
             {
-                Origin = Anchor.Centre,
-                Anchor = Anchor.Centre,
-            };
-
-            Add(buttonArea = new Container()
-            {
-                Anchor = Anchor.Centre,
-                Origin = Anchor.Centre,
-                SizeMode = InheritMode.X,
-                Size = new Vector2(1, button_area_height),
-                Alpha = 0
-            });
-
-            Add(osuLogo);
-
-            buttonArea.Add(new Box()
-            {
-                SizeMode = InheritMode.XY,
-                Colour = new Color4(50, 50, 50, 255)
-            });
-
-            buttonArea.Add(buttonFlow = new FlowContainerWithOrigin()
-            {
-                Anchor = Anchor.Centre,
-                Position = new Vector2(wedge_width * 2 - (button_width + osuLogo.SizeForFlow / 4), 0),
-                Padding = new Vector2(-wedge_width, 0)
+                osuLogo = new OsuLogo(onOsuLogo)
+                {
+                    Origin = Anchor.Centre,
+                    Anchor = Anchor.Centre
+                },
+                buttonArea = new Container()
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    SizeMode = InheritMode.X,
+                    Size = new Vector2(1, button_area_height),
+                    Alpha = 0,
+                    Children = new Drawable[]
+                    {
+                        new Box()
+                        {
+                            SizeMode = InheritMode.XY,
+                            Colour = new Color4(50, 50, 50, 255)
+                        },
+                        buttonFlow = new FlowContainerWithOrigin()
+                        {
+                            Anchor = Anchor.Centre,
+                            Position = new Vector2(wedge_width * 2 - (button_width + osuLogo.SizeForFlow / 4), 0),
+                            Padding = new Vector2(-wedge_width, 0),
+                            Children = new Drawable[]
+                            {
+                                settingsButton = new Button(@"settings", @"options", FontAwesome.gear, new Color4(85, 85, 85, 255), onSettings, -wedge_width, Key.O),
+                                backButton = new Button(@"back", @"back", FontAwesome.fa_osu_left_o, new Color4(51, 58, 94, 255), onBack, -wedge_width, Key.Escape),
+                                iconFacade = new Container() //need a container to make the osu! icon flow properly.

```

Comment:
Ok cool I'm getting more of an idea of who we're styling these, if we're using object initialisers we don't add parenthesis to the class name (unless we're passing args to the constructor)